debian/runtime-org.freedesktop.Platform.openh264.conf.in

```diff
@@ -0,0 +1 @@
+/usr/lib/@DEB_HOST_MULTIARCH@/openh264/extra
```

Comment:
Does GStreamer / the dynamic library loader "notice" immediately when the runtime gets installed that the symlink within this directory is no longer broken, or does something else have to re-run ldconfig?

Reply:
It seems to be the case here in my VM - after installing the pkgs I can immediately play the files with gst+playbin and sushi can preview them as well.

Reply:
Great – I wonder how that works.

Reply:
You don't have to rebuild the linker cache, but it's a good idea. It speeds up lookups, but it also ensures that the preferred library is loaded in case there's an existing one in the cache that's would be later in the search path. In this case there's likely no other `libopenh264.so.6` on the system. Still, it would be good to rebuild the cache after making modifications to the linker configuration.

I looked at the generated package and it indeed will trigger ldconfig because of the dummy library. However, it didn't actually include the directory or links because of:
```
/bin/sh: 1: lsb_release: not found
```
I think you should use `dpkg-vendor` for these kinds of checks. You could use `dpkg-vendor --query Vendor` and then use the string check or `dpkg-vendor --derives-from Endless && echo yes || echo no` or something like that.

Reply:
Thanks, good to know about ldconfig and related - fix for vendor check at https://github.com/endlessm/noopenh264/pull/5.